### PR TITLE
Add more sender data

### PR DESCRIPTION
**Pull request by nilgradisnik**
Store `sender_id`, `sender_name` and `sent_at` to database

### CODE REVIEW

File: aws/src/dynamo.js [hunk 1]
```diff
@@ -40,31 +41,22 @@ function messageCreated (webhook) {
   const messageBody = LayerIDK.getMessageText(message)
   if (!messageBody) return Promise.resolve()
 
-  const recipients = []
-  Object.keys(message.recipient_status).forEach((userId) => {
-    const status = message.recipient_status[userId]
-    if (status !== 'read') recipients.push(userId)
-  })
+  const recipients = Email.filterRecipients(message.recipient_status)
```
Comment (arunshan) [inline aws/src/dynamo.js]: Can we rename this to `filterReadRecipients` ?

File: common/email.js [hunk 2]
```diff
@@ -51,7 +62,9 @@ module.exports = class Email {
       try {
         hash[conversationId].push({
           id: Email.idPrefix('messages', item.message_id),
-          body: JSON.parse(item.message_body).text
+          body: JSON.parse(item.message_body).text,
+          sender_name: item.sender_name,
```
Comment (arunshan) [inline common/email.js]: I think you need to check if sender_name exists and use a default as we already have some existing users and `item.sender_name` can be null

File: common/email.js [hunk 3]
```diff
@@ -51,7 +62,9 @@ module.exports = class Email {
...
+          sent_at: item.sent_at
```
Comment (arunshan) [inline common/email.js]: Same here.

File: common/email.js (hunk 2 above)
Comment (nilgradisnik) [reply common/email.js]: So `sender_name` here is basically `display_name` on Identity object which can be null. I'm not sure what default we can use here. Do you mean something like `Unknown user`?

File: common/email.js (hunk 3 above)
Comment (nilgradisnik) [reply common/email.js]: Same fore `sent_at`? What do you mean.

File: common/email.js (hunk 2 above)
Comment (arunshan) [reply common/email.js]: But in the email template you are going to show undefined ?

**Pull request merged**